src/bin/cosmogony.rs

```diff
@@ -66,6 +67,37 @@ Accepted extensions are '.json', '.json.gz', '.jsonl', '.jsonl.gz'
     disable_voronoi: bool,
     #[structopt(help = "Only generates labels for given langs", long = "filter-langs")]
     filter_langs: Vec<String>,
+    #[structopt(
+        help = "File containing a list of bboxes to override",
+        long = "override-bbox-config"
+    )]
+    override_bbox_config: Option<PathBuf>,
```

Comment:
I have mixed feelings about this.  
On one hand, it's a very specific feature that we should definitely be able to desactivate.  
On the other hand, does it make sense to provide the exact configuration map on runtime, whereas the libpostal configuration is already embedded in the executable ?